File: Programmers/C#/lv1/lv1_평균 구하기.cs
https://programmers.co.kr/learn/courses/30/lessons/12944?language=csharp

public class Solution
{
    public double solution(int[] arr)
    {
        double answer = 0;

        int sum = 0;
        foreach (int n in arr)
        {
            sum += n;
        }
        answer = (double)sum / arr.Length;

        return answer;
    }
}

File: Programmers/lv1/lv1_평균 구하기.java
https://programmers.co.kr/learn/courses/30/lessons/12944
	
class Solution {
    public double solution(int[] arr) {
        double answer = 0;
        
        for(int i : arr){
            answer+=i;
        }
        answer = answer/arr.length;
        
        return answer;
    }
}